root/www/luci-static/resources/view/xray.js

```diff
@@ -136,10 +148,10 @@ return view.extend({
     render: function (load_result) {
         const config_data = load_result[0];
         const { geoip_existence, geoip_size, geosite_existence, geosite_size, optional_features } = check_resource_files(load_result[1]);
-        let asset_file_status = `WARNING: at least one of asset files (geoip.dat, geosite.dat) is not found under /usr/share/xray. Xray may not work properly. See <a href="https://github.com/yichya/luci-app-xray">here</a> for help.`
+        let asset_file_status = _('WARNING: at least one of asset files (geoip.dat, geosite.dat) is not found under /usr/share/xray. Xray may not work properly. See <a href="https://github.com/yichya/luci-app-xray">here</a> for help.')
         if (geoip_existence) {
             if (geosite_existence) {
-                asset_file_status = `Asset files check: geoip.dat ${geoip_size} bytes; geosite.dat ${geosite_size} bytes. Report issues or request for features <a href="https://github.com/yichya/luci-app-xray">here</a>.`
+                asset_file_status = _('Asset files check: geoip.dat ')+`${geoip_size}`+_(' ; geosite.dat ')+`${geosite_size}`+_(' . Report issues or request for features <a href="https://github.com/yichya/luci-app-xray">here</a>.')
```

Comment:
```suggestion
                asset_file_status = _('Asset files check: ') + `geoip.dat ${geoip_size}; geosite.dat ${geosite_size}. ` + _('Report issues or request for features <a href="https://github.com/yichya/luci-app-xray">here</a>.')
```

有些不需要翻译的最好就不要放在 `_()` 里面了。